### PR TITLE
fixed C347615

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js [hunk 1]
```diff
@@ -10,13 +11,28 @@ import SettingsMenu from '../../../support/fragments/settingsMenu';
...
+    incomingRecordType:NewFieldMappingProfile.incomingRecordType.edifact,
+    existingRecordType:NewFieldMappingProfile.folioRecordTypeValue.invoice,
+    description:'',
+    batchGroup: '"FOLIO"',
+    organizationName: NewFieldMappingProfile.organization.harrassowitz,
+    paymentMethod: '"Cash"'
+  };
+  const actionProfile = {
+    name: `Create Large Harrassowitz serials invoice ${getRandomPostfix()}`,
+    typeValue: 'Invoice',
```
Comment (penfriend) [inline cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js:29]: let's move this typeValue to the constants

File: cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js [hunk 2]
```diff
@@ -10,13 +11,28 @@ import SettingsMenu from '../../../support/fragments/settingsMenu';
...
+  const profileForDuplicate = FieldMappingProfiles.mappingProfileForDuplicate.harrassowitz;
+  const fileName = `C347615autotestFile.${getRandomPostfix()}.edi`;
+  const mappingProfile = {
+    name:`Import Large Harrassowitz Serials Invoice ${getRandomPostfix()}`,
+    incomingRecordType:NewFieldMappingProfile.incomingRecordType.edifact,
+    existingRecordType:NewFieldMappingProfile.folioRecordTypeValue.invoice,
+    description:'',
+    batchGroup: '"FOLIO"',
+    organizationName: NewFieldMappingProfile.organization.harrassowitz,
+    paymentMethod: '"Cash"'
```
Comment (penfriend) [inline cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js:25]: let's move this paymentMethod to the constants

File: cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js [hunk 3]
```diff
@@ -10,13 +11,28 @@ import SettingsMenu from '../../../support/fragments/settingsMenu';
...
-  const actionProfileName = `autoTestActionProf.${getRandomPostfix()}`;
-  const jobProfileName = `autoTestJobProf.${getRandomPostfix()}`;
+  const profileForDuplicate = FieldMappingProfiles.mappingProfileForDuplicate.harrassowitz;
+  const fileName = `C347615autotestFile.${getRandomPostfix()}.edi`;
+  const mappingProfile = {
+    name:`Import Large Harrassowitz Serials Invoice ${getRandomPostfix()}`,
+    incomingRecordType:NewFieldMappingProfile.incomingRecordType.edifact,
+    existingRecordType:NewFieldMappingProfile.folioRecordTypeValue.invoice,
+    description:'',
+    batchGroup: '"FOLIO"',
```
Comment (penfriend) [inline cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js:23]: let's move this batchGroup to the constants

File: cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js]: Done

File: cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js]: Done

File: cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/edi-import-large-invoice-file.cy.js]: Done

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ describe('ui-data-import', () => {
   const collectionOfMappingAndActionProfiles = [
     {
       mappingProfile: { name: `C380446 Test P/E mix open order with instance, holdings, item ${getRandomPostfix()}`,
-        typeValue : NewFieldMappingProfile.folioRecordTypeValue.order,
+        typeValue: FOLIO_RECORD_TYPE.ORDER,
         orderStatus: 'Open',
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:33]: this value of orderStatus can be moved to constant

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js [hunk 5]
```diff
@@ -45,24 +45,24 @@ describe('ui-data-import', () => {
...
         permanentLocationUI:'Main Library' },
-      actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.holdings,
+      actionProfile: { typeValue: FOLIO_RECORD_TYPE.HOLDINGS,
         name: `C380446 Create simple holdings for open order ${getRandomPostfix()}` }
     },
     {
       mappingProfile: { name: `C380446 Create simple item for open order ${getRandomPostfix()}`,
-        typeValue : NewFieldMappingProfile.folioRecordTypeValue.item,
+        typeValue: FOLIO_RECORD_TYPE.ITEM,
         materialType:'book',
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:62]: this value can be taken from the constant

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js [hunk 6]
```diff
@@ -45,24 +45,24 @@ describe('ui-data-import', () => {
...
         permanentLoanType: 'Course reserves',
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:63]: this value can be moved to constant

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js [hunk 7]
```diff
@@ -45,24 +45,24 @@ describe('ui-data-import', () => {
...
         status: 'On order' },
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:64]: this value can be moved to constant

File: cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js [hunk 8]
```diff
@@ -78,7 +78,7 @@ describe('ui-data-import', () => {
         temporaryLoanType:'"Course reserves"',
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js:78]: this value can be moved to constant

File: cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js [hunk 9]
```diff
@@ -78,7 +78,7 @@ describe('ui-data-import', () => {
...
         temporaryLoanTypeUI:'Course reserves',
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js:79]: this value can be moved to constant

File: cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js [hunk 10]
```diff
@@ -78,7 +78,7 @@ describe('ui-data-import', () => {
...
         status: 'Available' },
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js:80]: this value can be taken moved to constant

File: cypress/e2e/data-import/log-details/import-summary-table-for-creating-actions.cy.js [hunk 11]
```diff
@@ -33,25 +33,25 @@ describe('ui-data-import', () => {
...
         pernanentLocationUI:'Online' },
-      actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.holdings,
+      actionProfile: { typeValue: FOLIO_RECORD_TYPE.HOLDINGS,
         name: holdingsActionProfileName }
     },
     {
-      mappingProfile: { typeValue: NewFieldMappingProfile.folioRecordTypeValue.item,
+      mappingProfile: { typeValue: FOLIO_RECORD_TYPE.ITEM,
         name: itemMappingProfileName,
         permanentLoanType: 'Can circulate',
```
Comment (penfriend) [inline cypress/e2e/data-import/log-details/import-summary-table-for-creating-actions.cy.js:52]: this value can be moved to constant

File: cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js (hunk 9 above)
Comment (penfriend) [reply cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js:79]: it is fixed in the pr 1133

File: cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js (hunk 8 above)
Comment (penfriend) [reply cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js:78]: it is foxed in pr 1133

File: cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js (hunk 10 above)
Comment (penfriend) [reply cypress/e2e/data-import/importing-marc-bib-files/test-remove-field-mapping-option.cy.js:80]: it is fixed in pr 1133

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js (hunk 6 above)
Comment (penfriend) [reply cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:63]: it is fixed in pr 1133

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js (hunk 5 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:62]: it is fixed in pr 1133

File: cypress/e2e/data-import/log-details/import-summary-table-for-creating-actions.cy.js (hunk 11 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/log-details/import-summary-table-for-creating-actions.cy.js:52]: it is fixed in pr 1133

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:33]: Done

File: cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js (hunk 7 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/mrc-import-create-open-order-pe-mix-with-instance-holdings-items.cy.js:64]: Done

**Review by penfriend — APPROVED** [review]

**Pull request merged**